deploy/helm/kubecf/assets/operations/instance_groups/routing-api.yaml

```diff
@@ -28,3 +31,35 @@
 {{- range $path, $bytes := .Files.Glob "assets/operations/pre_render_scripts/routing-api_*" }}
 {{ $root.Files.Get $path }}
 {{- end }}
+
+{{- else }}
+# Disable routing-api.
+- type: remove
+  path: /instance_groups/name=api/jobs/name=routing-api
+
+- type: remove
+  path: /instance_groups/name=database?/jobs/name=pxc-mysql/properties/seeded_databases/name=routing-api
```

Comment:
@mook-as you can do that, but independently of the DB type in KubeCF, you need to remove this, while it is a property that lives directly in https://github.com/cloudfoundry/cf-deployment/blob/v12.33.0/cf-deployment.yml#L375-L377

Reply:
Ah! I got confused by the `name=database?`; looking at [`database.yaml`](https://github.com/cloudfoundry-incubator/kubecf/blob/master/deploy/helm/kubecf/assets/operations/instance_groups/database.yaml#L2) we actually unconditionally remove the instance group. We do have a separate db-seeder that you've already covered in this PR.

That is, this is just a long-winded way to ask: why do we need the `?` on that path? Is it becase that instance group has _already_ been removed, and there's no point in removing it again?

Reply:
yes

Reply:
In that case, shouldn't we drop the whole operation (since it's, essentially, removing something that doesn't exist)?

Reply:
@mook-as sorry, I misunderstood your question. The `?` was just in case the DB was removed, we removed the DB in KubeCF in https://github.com/cloudfoundry-incubator/kubecf/blob/master/deploy/helm/kubecf/assets/operations/instance_groups/database.yaml#L2-L3 , but the `?` needs to be there as a security measure, unless you know the interpolation order when applying the different YAMLS from https://github.com/cloudfoundry-incubator/kubecf/tree/master/deploy/helm/kubecf/assets/operations/instance_groups on top of the cf-deployment manifest. In other words, how to know what happens first, https://github.com/cloudfoundry-incubator/kubecf/blob/master/deploy/helm/kubecf/assets/operations/instance_groups/routing-api.yaml vs https://github.com/cloudfoundry-incubator/kubecf/blob/master/deploy/helm/kubecf/assets/operations/instance_groups/routing-api.yaml ?

Reply:
Ah! I thought we sorted the glob, but we don't, so we'll have to do the guard. Thanks!